ecl/hql/hqlattr.cpp

```diff
@@ -308,6 +308,7 @@ unsigned getOperatorMetaFlags(node_operator op)
     case no_normalize:
     case no_distributed:
     case no_preservemeta:
+    case no_unordered:
```

Comment:
minor: for future maintenance and aiding intuition of newbies, does it makes sense to remove the unorderdAtom and an orederedAtom, yet create this node att as unordered? Does it make sense have the semantic logic of the atom and the node att/type to be the inverse of one another?